src/Components/Shifting/ListView.tsx

```diff
@@ -292,12 +303,13 @@ export default function ListView() {
               </button>
             </div>
 
-            <div className="mb-5 grid gap-x-6 md:grid-cols-2">
-              {showShiftingCardList(shiftData?.results || [])}
-            </div>
-            <div>
-              <Pagination totalCount={shiftData?.count || 0} />
-            </div>
+            {showShiftingCardList(shiftData?.results || [])}
+
+            {(shiftData?.results?.length ?? 0) > 0 && (
+              <div>
+                <Pagination totalCount={shiftData?.count || 0} />
+              </div>
+            )}
```

Comment:
Is this check needed? Pagination component hides itself if the count is 0

<img width="667" alt="image" src="https://github.com/user-attachments/assets/8160efac-2bf2-4272-b4e9-373f7e7fe7b3">

Reply:
Yes, that check is not required, it's working as expected without that check. I'll remove it